.github/workflows/ci.yml

```diff
@@ -228,7 +228,7 @@ jobs:
   test_integration:
     name: Integration Tests
     runs-on: ubuntu-latest
-    timeout-minutes: 25
+    timeout-minutes: 15
```

Comment:
Was this 15 or 20?

Reply:
`15`

Reply:
Bumped this back to `20`,  since after running this just and I noticed that if we had a clean build it would take up to 17 min. So let's stay on the safe side and have some room in case we will have to re-build the tree from scratch.